.github/workflows/ci.yaml

```diff
@@ -0,0 +1,48 @@
+on: [push]
```

Comment:
I ended up doing `push` and `pull_request` on https://github.com/googleapis/repo-automation-bots/pull/261/files, do you think we could get away with just `push`?

Reply:
`pull_request` might imply `push`, but I'm not sure it does. It probably doesn't hurt to include it.